### DISCUSSION TITLE
How can I use class that I've coded with winUI in wpf project?

### DISCUSSION THREAD

**Discussion by samsunlu55atakum** [microsoft/microsoft-ui-xaml#8790] [Q&A]
I want to use control or elements(xaml) that I've coded with winUI in WPF app. How can I do that? Can you describe step-by-step?

**Comment by DarranRowe** [comment]
For now, I think it is better for you to wait.
The answer to this is Xaml Islands, but as the [1.4 Preview 2 release notes state](https://learn.microsoft.com/en-gb/windows/apps/windows-app-sdk/preview-channel#xaml-islands-no-longer-experimental):
"Currently XAML Islands are only tested for use in C++ apps. This release does not include any convenient wrapper elements for use in WPF or WinForms."
Hopefully there will be documentation and samples available that could answer your question when 1.4 is properly released.
